### ISSUE TITLE
training results 

### ISSUE THREAD

**Issue by JAYCHOU2020** [JCruan519/VM-UNet#45]
Hi, thank you for your work while I was training at 4090 Single card, raining isic18 with the pretraining model pre_trained_weights/vssmsmall_dp03_ckpt_epoch_238.pth could not get the miou result of 81.35 in the paper.Only 0.793 miou

**Comment by fangxiao-yu** [comment]
Hello, can your loss go down? My loss can't go down. I don't know why

**Comment by Aaronwangz** [comment]
/vssmsmall_dp03_ckpt_epoch_238.pth，Was this weight found in vmanba? At that specific location?

**Comment by anran0422** [comment]
@JAYCHOU2020 你好，我通过查看代码的模型加载路径，发现不动代码就是加载了预训练模型 e238.pth，但是我在isic2018数据集上的结果并不像论文中结果那样。
我的疑惑在于，你更改代码了吗，为什么你的miou能打到0.793，我的是只有0.57（测试集），付一张图片
![isic2018](https://github.com/user-attachments/assets/5cb62da1-3d50-4128-aba7-3ef389851f96)

**Comment by anran0422** [comment]
@Aaronwangz 我在特定位置，百度网盘下载的，大致像这个名字，但不是

**Comment by lhhvictory** [comment]
你训练速度慢吗？batch_size设为2,为什么我训练一个batch需要37秒，这么慢？？

**Comment by anran0422** [comment]
@lhhvictory 就是最默认的运行，其他的都没有动
